lib/index.js

```diff
@@ -292,17 +304,36 @@ class Embark {
           callback();
         }
       },
-      function buildAndDeployContracts(callback){
+      function startServices(callback) {
+        let pluginList = engine.plugins.listPlugins();
+        if (pluginList.length > 0) {
+          engine.logger.info("loaded plugins: " + pluginList.join(", "));
+        }
+
+        engine.startService("libraryManager");
+        engine.startService("web3");
+        engine.startService("pipeline");
+        engine.startService("codeGenerator");
+        engine.startService("deployment");
+        engine.startService("ipfs");
```

Comment:
he suggestion is:
1. add here ```engine.startService("swarm")```

and add the relevant code to engine.js (see ipfs e.g)

Reply:
Updated

Reply:
👍